Mathlib/Data/Nat/Factorization/Defs.lean

```diff
@@ -0,0 +1,288 @@
+/-
+Copyright (c) 2021 Stuart Presnell. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Stuart Presnell
+-/
+import Mathlib.Data.Finsupp.Multiset
+import Mathlib.NumberTheory.Padics.PadicVal
```

Comment:
It would be nice if we could further avoid this for the Defs file, but this is already a great improvement so

bors d+

Reply:
I'm not sure if there is a file that would benefit from it. I'll have a look for a possible second iteration.